tests/sapt-exch-ind-inf/input.dat

```diff
@@ -0,0 +1,36 @@
+#! SAPT(DFT) aug-cc-pVDZ interaction energy between Ne and Ar atoms.
+
+Eref = {'Exch-Ind20,u (A<-B)':          0.00749283, # TEST
+        'Exch-Ind20,u (A->B)':          0.07674835, # TEST
+        'Exch-Ind20,u':                 0.08424119, # TEST
+        'Exch-Ind20,u (A<-B) (Sinf)':   0.00749701, # TEST
+        'Exch-Ind20,u (A->B) (Sinf)':   0.07677809, # TEST
+        'Exch-Ind20,u (Sinf)':          0.08427511, # TEST
+        'Exch-Ind20,r (A<-B)':          0.00838528, # TEST
+        'Exch-Ind20,r (A->B)':          0.08663184, # TEST
+        'Exch-Ind20,r':                 0.09501712, # TEST
+        'Exch-Ind20,r (A<-B) (Sinf)':   0.00839011, # TEST
+        'Exch-Ind20,r (A->B) (Sinf)':   0.08666563, # TEST
+        'Exch-Ind20,r (Sinf)':          0.09505574} # TEST
+
+
+molecule dimer {
+  Ne
+  --
+  Ar 1 6.5
+  units bohr
+}
+
+set {
+    basis         aug-cc-pvdz
+    scf_type      df
+    sapt_dft_grac_shift_a 0.203293
+    sapt_dft_grac_shift_b 0.138264
+    Sinf          True
```

Comment:
This shouldn't work as shown, can you remove the `Sinf` option here and change it for the one presented in the comments? Also can delete the GRAC shifts, they are not valid for SAPT(HF).